cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -21,7 +21,14 @@
     {%- endfor %}
 
     {% for block in page.content -%}
-        {{ render_block.render(block, loop.index) }}
+        {% if block.block_type == 'feedback' %}
```

Comment:
Yeah, I think it'd be nice if the logic was included in a partial. I don't think it's out of scope since the duplication is happening within this PR.